.api/v1.4.0/PowerBI-visuals.d.ts

```diff
@@ -0,0 +1,1282 @@
+/*
```

Comment:
I don't think that we should include an API to the repo.
I think it'd be better to [install API automatically](https://github.com/Microsoft/PowerBI-visuals-ChicletSlicer/blob/master/package.json#L14).